config/envoyconfig/routes.go

```diff
@@ -102,6 +98,7 @@ func (b *Builder) buildPomeriumAuthenticateHTTPRoutes(
 			return []*envoy_config_route_v3.Route{
 				b.buildControlPlanePathRoute(options, options.AuthenticateCallbackPath),
 				b.buildControlPlanePathRoute(options, "/"),
+				b.buildControlPlanePathRoute(options, "/robots.txt"),
```

Comment:
Why does the authenticate service need a robots.txt route?

Reply:
Because routes to authenticate bypass the authorize service. Without this exception the robots.txt route will return a 404.